ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift

```diff
@@ -5,11 +5,61 @@
 
 import UIKit
 
-class ShimmerLinesViewDemoController: DemoController {
-    override func viewDidLoad() {
-        super.viewDidLoad()
+class ShimmerViewDemoController: DemoController {
+	override func viewDidLoad() {
+		super.viewDidLoad()
 
-        // Default Shimmer
-        container.addArrangedSubview(ShimmerLinesView())
-    }
+		let contentView = { () -> UIStackView in
+			let label1 = UILabel()
```

Comment:
Label() instead of UILabel() to make sure larger text size ....would also increase the shimmer re-sizes correctly?

Reply:
Nice catch =). 

![Screen Shot 2020-09-08 at 9 57 20 AM](https://user-images.githubusercontent.com/6722175/92506042-b5d37780-f1b9-11ea-970d-6de4ee6a1712.png)